contributor/making-release.md

```diff
@@ -22,6 +22,14 @@ There are number of things that are required before attempting to build a releas
 
 Given all of this, it's recommended that you only attempt making a release from a GNU/Linux machine.
 
+## Triage issues.
+
+Before creating a release candidate, all open issues with a fix version of the
+release candidate should be triaged.  During the transition from JIRA to GitHub
+issues, anyone doing triage will need to search for open issues in both
+locations.  Searching in both locations is only necessary for branches that
+have not released since the transition started.
```

Comment:
An absence of open issues with the fix version label.

Reply:
If you are releasing 2.0.0, search for open issues with the labels `v2.0.0` and `blocker`. If there are none, you can release. Ideally, we would fix/close all open issues with `v2.0.0` but I don't think that is ever possible.  After release, remaining open issues can be given `v2.1.0` label and `v2.0.0` label can be removed.

Reply:
Keith just pointed me at https://github.com/apache/accumulo/pull/396/files#diff-6a3371457528722a734f3c51d9238c13R35 which explains the `vX.Y.Z` convention for tracking "fixVersion" on GH issues.

At risk of duplicating information across these two PRs, should there be some consolidation here?

Reply:
> At risk of duplicating information across these two PRs, should there be some consolidation here?

Yes, I think it would be best to move the info in contrib.md to the website and mention that the labels only apply to the main repo.  Could remove the info from contrib.md.

Reply:
My thinking was to document how each repo uses labels in Github issues in contrib.md, to allow for different labels.  We could do this in future, but for now I think the labels in the core repo should be the focus.

Reply:
> Are we using the milestones feature?

I really wish we could make use of milestones, but there does not seem to be a good way available to have multiple milestones. The only way I could think of was to have multiple issues, with different milestones, for the same issue, but I don't think that's as convenient as labels. It would be confusing to mix and match the two, though we may discover a good way to use them in the future (milestones based on time rather than versions, for example). I think Keith's idea to use labels is better.

Reply:
> Yes, I think it would be best to move the info in contrib.md to the website and mention that the labels only apply to the main repo. Could remove the info from contrib.md.

Did this happen?

Reply:
Yes, in 5b0d6c4

Reply:
Ok, thanks!